### PR TITLE
add archived indicator to projects

### PR DESCRIPTION
**Pull request by macbookandrew**
## Description

Add an archived status indicator:

Screen Shot 2022-08-31 at 15 41 03

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Wouldn't it be better to move stars and archives to [accessories](https://developers.raycast.com/api-reference/user-interface/list#list.item), then you can add icons too?

**Comment by pernielsentikaer** [comment]
Can you check this out @tonka3000

**Comment by tonka3000** [comment]
@pernielsentikaer i will check it in the evening

**Comment by Loskir** [comment]
@macbookandrew could you send new screenshots please?

**Comment by Loskir** [comment]
I don't quite like that archived projects are indicated with a red exclamation mark, maybe we should use secondary color instead?

**Comment by metakirby5** [comment]
Agree with @Loskir, and I think `Icon.Tray` may be more fitting.

image

**Comment by tonka3000** [comment]
I also would prefer an archived icon with e.g. yellowish color. The red one is very "heavy" and the color indicate something "bad".

CleanShot 2022-09-05 at 15 06 46@2x

**Comment by Loskir** [comment]
Hmm, why are the stars not aligned?

**Comment by macbookandrew** [comment]
> Agree with @Loskir, and I think `Icon.Tray` may be more fitting.
> 
> image

I’m not opposed to the tray, though gitlab itself does use an exclamation mark

67842476-CF75-417C-9AEF-6B54F5437452

**Comment by andrewminion-luminfire** [comment]
@tonka3000 thanks for the tooltip suggestion. Updated with yellow and tooltip:

Screen Shot 2022-09-07 at 12 58 38

File: extensions/gitlab/src/components/project_search.tsx [hunk 1]
```diff
@@ -22,11 +22,16 @@ import { ClearLocalCacheAction } from "./cache_actions";
 
 export function ProjectListItem(props: { project: Project }): JSX.Element {
   const project = props.project;
+  const accessories = [];
+  if (project.archived) {
+    accessories.push({ text: "Archived", icon: { source: Icon.ExclamationMark, tintColor: Color.Red } });
```
Comment (metakirby5) [inline extensions/gitlab/src/components/project_search.tsx:27]: @macbookandrew @andrewminion-luminfire Looks like it still has the text and is red, maybe forgot to push a commit?

Comment (macbookandrew) [reply extensions/gitlab/src/components/project_search.tsx:27]: 🤦🏻 yes, I forgot to push the commit. #2837 includes that change. Sorry!!

Comment (macbookandrew) [reply extensions/gitlab/src/components/project_search.tsx:27]: already did…see #2837

**Pull request merged**